low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1.yaml

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+# Version 1.0.0
+
+SYSTEM_INFORMATION:
+    cli_type: CS_A
+    data_path: /var/cortx/sspl/data/
+    log_level: INFO
+    sspl_log_file_path: /var/log/cortx/sspl/sspl.log
+    syslog_host: localhost
+    syslog_port: 514
+
+RABBITMQINGRESSPROCESSOR:
+    virtual_host: SSPL
+    queue_name: actuator-req-queue
+    exchange_name: sspl-in
+    routing_key: actuator-req-key
+    username: sspluser
+    password:
```

Comment:
In documentation, make sure we are capturing to add encrypted passwords, with steps to encrypt also